contents/tutorials/rss-item-capture.md

```diff
@@ -0,0 +1,122 @@
+---
+title: How to capture new RSS items in PostHog (releases, blogs, status)
+date: 2023-08-29
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-6.png
+tags: ['events']
+---
+
+RSS is a popular format for providing feeds of content. For example, GitHub uses it to provide feeds of releases, blogs provide feeds of new posts, and status pages provide feeds of incidents
+
+We can capture events from these feeds by polling them, checking for new entries, and then capturing them into PostHog. This is exactly what we do in this tutorial, with the help of [Val Town](https://www.val.town/), a platform for writing, running, and scheduling JavaScript functions in your browser. 
+
+## Creating our capture function in Val Town
+
+After signing up for [Val Town](val.town), you can go to your workspace and start creating JavaScript functions. We start by writing a function to capture an event using the [PostHog API](/docs/api).
+
+Since we will use this function to capture events from multiple feeds, we make it generic. We write it as a function that takes an object ccontaining a `key`, `event`, `properties`, and `distinct_id`. It uses these to create the body of the request and then send a fetch request to `https://app.posthog.com/capture/` (or `https://eu.posthog.com/capture/`). Altogether, this looks like this:
```

Comment:
```suggestion
Since we will use this function to capture events from multiple feeds, we make it reusable. We write it as a function that takes an object ccontaining a `key`, `event`, `properties`, and `distinct_id`. It uses these to create the body of the request and then send a fetch request to `https://app.posthog.com/capture/` (or `https://eu.posthog.com/capture/`). Altogether, this looks like this:
```